can_tools/scrapers/official/SD/sd_vaccines.py

```diff
@@ -99,34 +99,6 @@ def construct_body(self, resource_key, ds_id, model_id, report_id, counties):
                     }
                 }
             },
-            {
```

Comment:
So, we actually need these booster shots to calculate the other doses correctly. 

SD defines each bucket as the number of people who currently have the bucket-specific number of doses. So someone in the "1-dose" Moderna bucket only has 1 dose, and when they receive their second dose they are removed from this bucket and added to the "2-dose" bucket. 

The same goes for boosters, so we need to add the booster shots to the existing dose buckets to get accurate fully vaccinated and fully initiated counts.

Reply:
Also, FWIW the dashboard looks like it has some problems. It's displaying negative values for the manufacturer-specific data. The underlying data (that we access) seems to be okay, but it is a little concerning.

![image](https://user-images.githubusercontent.com/55333380/140170723-3563016e-4241-402c-99bd-9baa4dede48f.png)

Reply:
Oh, wow.  That's tricky / annoying.  Nice work detangling this.